tasks/audit-cat1.yml

```diff
@@ -103,6 +103,8 @@
       - RHEL-07-040331
       - always
 
+- debug: var={{ rhel_07_040331_audit }}
```

Comment:
Not sure if this is supposed to be here. If it is, please see the guidelines for proper task naming and do not Ansible shorthand syntax.